lib/Rex/Interface/Exec/Local.pm

```diff
@@ -119,6 +119,11 @@ sub exec {
 
   }
 
+  # we need to bitshift $? so that $? contains the right (and for all
+  # connection methods the same) exit code after a run()/i_run() call.
+  # this is for the user, so that he can query $? in his task.
+  $? >>= 8;
```

Comment:
It seems like a bad idea to modify $?.  How will users no its not the usual unbitshifted value that Perl normally provides.  Why can't the user bitshift it 8 themselves if they want that value.